iac-extensions/docker/src/main/java/org/sonar/iac/docker/checks/utils/CommandDetector.java

```diff
@@ -253,6 +259,21 @@ public CommandDetector.Builder withPredicatesFrom(CommandDetector.Builder otherB
       return this;
     }
 
+    public CommandDetector.Builder withIncludeUnresolved(Predicate<String> predicate) {
+      addIncludeUnresolved(predicate, MATCH);
+      return this;
+    }
+
+    public CommandDetector.Builder withAnyExcludingIncludeUnresolved(Predicate<String> predicate) {
```

Comment:
The problem in my opinion is the hard brake between `Excluding` and `Include`.
Previously we had the excluding at the end so that it's related to the parameter provided. My suggestion would be:
`withAnyIncludingUnresolvedExcluding` or to be more verbose:
`withAnyIncludingUnresolved(But|And)Excluding`